crates/extricrate/src/lib.rs

```diff
@@ -460,6 +521,31 @@ pub mod dependencies {
             );
         }
 
+        #[test]
+        fn build_a_simple_dependency_map() {
```

Comment:
Can you add a test where we import several things from the same module?

`use module_a::{Foo, Bar};`